packages/itwin/ec3-widget/src/components/EC3/generated/api.ts

```diff
@@ -0,0 +1,712 @@
+/*---------------------------------------------------------------------------------------------

```

Comment:
Do not add a generated client. I guess it would be better to write a non-generated client here that could be copied over to the insights-client repo later. This generated client adds a lot of information and complexity that is not really needed here.

Reply:
generated folder was redundant and the files were removed